sqe-http-api/Controllers/ArtefactController.cs

```diff
@@ -57,7 +57,7 @@ public async Task<ActionResult<ArtefactListDTO>> GetArtefacts([FromRoute] uint e
         /// <param Name="ArtefactId">Unique Id of the desired artefact</param>
         /// <param Name="images">Defines whether image references should be included in the response</param>
         [AllowAnonymous]
-        [HttpGet("editions/{editionId}/artefacts/{ArtefactId}")]
+        [HttpGet("editions/{editionId}/artefacts/{artefactId}")]
```

Comment:
Somehow the route parameter artefactId was changed to ArtefactId by my fixing of the classes to use PascalCase.  I fix it here.